instrumentation-docs/readme.md

```diff
@@ -10,8 +10,8 @@ Run the doc generator:
 
 ## Instrumentation Hierarchy
 
-An "InstrumentationEntity" represents a module that that targets specific code in a framework/library/technology.
-Each instrumentation uses muzzle to determine which versions of the target code it supports.
+An "InstrumentationEntity" represents a module that that targets specific code in a
```

Comment:
if you think of another word besides entity, that might be good since we have https://github.com/open-telemetry/opentelemetry-specification/pull/4442